--- appwork-utils-0.0+git20140605~887432732f/src/org/appwork/utils/swing/windowmanager/WindowManager.java	2014-06-05 05:03:25.000000000 -0430
+++ appwork-utils-0.0+git20140605~887432732f~/src/org/appwork/utils/swing/windowmanager/WindowManager.java	2015-06-16 15:38:25.506351900 -0430
@@ -56,7 +56,7 @@
         } else if (CrossSystem.isLinux()) {
             return new LinuxWindowManager();
         } else if (CrossSystem.isMac()) {
-            return new MacWindowManager();
+            return new DefaultWindowManager();
         } else {
             return new DefaultWindowManager();
         }
